cppclean

```diff
@@ -88,6 +88,17 @@ def main():
                         help='add a header include path; '
                              'specify this multiple times for multiple '
                              'include paths')
+    parser.add_argument('--include-path-system', '-is', '-Is', action='append',
+                        dest='include_system_paths', default=[],
```

Comment:
Could you add a `metavar='path'` like in the `--include-path` option. Otherwise, the `--help` gets a bit messy:

```
  --include-path path, -i path, -I path
                        add a header include path; specify this multiple times
                        for multiple include paths
  --include-path-system INCLUDE_SYSTEM_PATHS, -is INCLUDE_SYSTEM_PATHS, -Is INCLUDE_SYSTEM_PATHS
                        same as --include-path but explicitly designates all
                        header files found in thesedirectories as "system"
                        includes
  --include-path-non-system INCLUDE_NONSYSTEM_PATHS, -in INCLUDE_NONSYSTEM_PATHS, -In INCLUDE_NONSYSTEM_PATHS
                        same as --include-path but explicitly designates all
                        header files found in thesedirectories as "non-system"
                        includes
```